server.js

```diff
@@ -24,12 +25,89 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+if (process.env.RESET_DB) {
+  const resetDatabase = async () => {
+    await Book.deleteMany({});
+    booksData.forEach(item => {
+      const newBook = new Book(item)
+      newBook.save()
+    })
+  }
+  resetDatabase()
+}
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+res.send("This is an api for books.");
```

Comment:
One way to do it is to add a guide with all the endpoints- It will be easer to know which endpoints exists without looking in the code.